student-software-developers/administration-handbook/timeline/05-may.md

```diff
@@ -0,0 +1,7 @@
+### Late May / Early June: First day
+
```

Comment:
We should add a link to https://github.com/pulibrary/pul-it-handbook/blob/main/student-software-developers/student-handbook.md#day-1-kick-off-meeting -- I merged it so we should be able to do that now.